tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -721,7 +722,7 @@ SPECTRE_TEST_CASE("Unit.ApparentHorizons.StrahlkorperGr.SpinVector",
       Strahlkorper<Frame::Inertial>(l_max, l_max, get(horizon_radius), center);
 
   const auto horizon_radius_with_spin_on_z_axis =
-      TestHelpers::Kerr::horizon_radius(
+      gr::Solutions::kerr_horizon_radius(
           Strahlkorper<Frame::Inertial>(l_max, l_max, 2.0, center)
               .ylm_spherepack()
               .theta_phi_points(),
```

Comment:
Can you test that the horizon is correct at a couple of points for Kerr Schild? If that’s a pain, maybe test that it gives a sphere for zero spin?